web/styles.css

```diff
@@ -278,13 +298,107 @@ body {
 }
 
 .selected-link {
-    color:#ffffff;
+    color: #ffffff;
     font-weight: bold;
     border-bottom: 3px solid #347d39;
 }
 
+.button {
+    transition: all 0.05s;
+    display: inline-block;
+    padding: 10px 20px;
+    background-color: transparent;
+    outline: solid 1px grey;
+    color: dimgrey;
+    font-weight: bold;
+    text-decoration: none;
+    border-radius: 5px;
+    margin: 20px;
+    cursor: pointer;
+    width: fit-content;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.button:hover {
+    outline: solid 1px #d6d6d6;
+    color: #d6d6d6;
+}
+
+.draggable {
+    display: inline-block;
+    padding: 10px 20px;
+    background-color: #347d39;
+    color: #FFFFFF;
+    text-decoration: none;
+    border-radius: 5px;
+    cursor: move !important;
+    width: fit-content;
+}
+
+.dragging {
+    opacity: 0.001;
+}
+
+.draggable-wrapper {
+    overflow: visible;
+    box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    outline: 2px solid transparent;
+    display: inline-block;
+    width: fit-content;
+    border-radius: 5px;
+    padding: 0px;
+    margin: 20px;
+}
+
+.show-outline {
+    outline: 2px dotted #c7d5e580;
+}
+
+@keyframes shake {
```

Comment:
This is the shake animation that the bookmarklet plays when it is clicked instead of dragged